py-rse/bash-advanced.Rmd

```diff
@@ -254,36 +258,40 @@ $ cat summarize_all_books.sh
 302 history 6 > summarize_all_books.sh
 ```
 
-Open the file in an editor,
+We can now open the file in an editor,
```

Comment:
Isn't this still a hypothetical?
```suggestion
We could now open the file in an editor,
```
Which begs the question, why don't we just get the learner to run those six lines, then it doesn't have to be hypothetical...

Reply:
Actually later output includes `authors.txt`, `releases.txt`, and `summarize_all_books.sh`, so we should definitely make learners run these commands.

Reply:
I think I've fixed this - can you please check?